src/Backend/Modules/Location/Installer/Installer.php

```diff
@@ -19,6 +21,8 @@ public function install(): void
         $this->configureBackendNavigation();
         $this->configureBackendRights();
         $this->configureFrontendExtras();
+
+        Model::get('fork.entity.create_schema')->forEntityClass(Location::class);
```

Comment:
Can you please create a method `$this->configureEntities();` and move your code into that new method.
You best add that method call just below the `importLocale(...)`. Then it is in line with all other modules in fork cms core.